index.html

```diff
@@ -758,26 +758,361 @@ <h2>Extensions</h2>
+that:
+        </p>
+
+        <pre class="example nohighlight" title="Example schema.org address with full URLs">
+          {
+            "@type": "http://schema.org/Person",
+            "http://schema.org/address": {
+              "@type": "http://schema.org/PostalAddress",
+              "http://schema.org/streetAddress": "123 Main St."
+              "http://schema.org/addressLocality": "Blacksburg",
+              "http://schema.org/addressRegion": "VA",
+              "http://schema.org/postalCode": "24060",
+              "http://schema.org/addressCountry": "US"
+            }
+          }
+        </pre>
+
+        <p>
+While this form is an acceptable way to express the information such that it is
+unambiguous, many software developers would prefer to use more idiomatic JSON.
+The use of <code>@context</code> enables idiomatic JSON without losing global
+uniqueness and without the need for a centralized registry or authority for
+creating extensions. Note that <code>@context</code> can also have more than
+one value. In this case, a JSON array is used to express multiple values, where
+each value references another context that defines terms. Using this mechanism
+we can first bring in the terms defined in the Verifiable Credentials Data Model
+specification and then bring in the terms defined by schema.org:
+        </p>
+
+        <pre class="example nohighlight" title="Example address credential with schema.org context">
+          {
+            <span class="highlight">
+            "@context": [
+              "https://www.w3.org/credentials/v1",
+              "http://schema.org"
+            ]</span>,
+            ...
+            "credentialSubject": {
+              "type": "Person",
+              "address": {
+                "type": "PostalAddress",
+                "streetAddress": "123 Main St."
+                "addressLocality": "Blacksburg",
+                "addressRegion": "VA",
+                "postalCode": "24060",
+                "addressCountry": "US"
+              }
+            },
+            ...
+          }
+        </pre>
+
+        <p>
+Note, however, that each <em>context</em> might have a different definition for
+the same term, e.g., the JSON key <code>address</code> might map to a different
+URL in each <em>context</em>. By default, [JSON-LD] allows terms in a
+<code>@context</code> to be redefined using a <em>last term wins</em>
+order. While these changes can be safely dealt with by using JSON-LD
+processing, we want to lower the burden on consumers of Verifiable Credentials.
+We want consumer software to be able to make assumptions about the meaning of
+terms by only having to read and understand the string value associated with
+the <code>@context</code> key. We don't want them to have to worry about terms
+being redefined in unexpected ways. That way their software can inspect only
+the <code>@context</code> values and then be hard coded to understand the
+meaning of the terms.
```

Comment:
The paragraph above *requires* immutability of the `@context` definition. schema.org doesn't measure up.